script/generate-octicons.ts

```diff
@@ -46,7 +46,7 @@ async function generateIconData(): Promise<ReadonlyArray<IOcticonData>> {
 
   const results = new Array<IOcticonData>()
 
-  for (const name of Object.keys(octicons)) {
+  for (const name of Object.keys(octicons).sort()) {
```

Comment:
I added this step to ensure the generated file is sorted alphabetically - the latest package has a different order after doing `const octicons = require('octicons')`